docs/Selectors/Link selector.md

```diff
@@ -10,12 +10,16 @@ to. If you are selecting multiple links then check *multiple* property.
 using AJAX for data loading. Instead of using a link selector, you
 should use [Pagination selector] [pagination-selector].
 
-The *Link selector* can extract links from 4 types of sources:
+The *Link selector* can extract links from 5 types of sources:
 
 1. **Link** - reads the `href` attribute of an element. E.g. `<a href="https://example.com">`;
 2. **Text** - reads the text content of an element. E.g. `<span>https://example.com</span>`;
 3. **Attribute** - reads the attributes of an element and finds the link. E.g. `<a data-link="https://example.com">`;
-4. **Script** - reads the scripted link in an attribute. E.g. `<a onclick="window.location='https://example.com'">`;
+4. **Scripted link in attribute** - reads the scripted link in an attribute. E.g. `<a onclick="window.location='https://example.com'">`;
+5. **Link from any script** - reads link from a script. E.g. `<a(window.location=, window.open)>`;
+
+The Link Selector is only recognizing <a> elements using the point-and-click interface to be a selector, 
```

Comment:
All Link Selector types except for 'Link from any script' only allow selecting <a> elements using the point-and-click interface
although other elements can be selected by manually entering the CSS selector value as a selector.

The 'Link from any script' type allows any element to be selected using the point-and-click interface.